client/src/views/admin-academy/CoursesDetail.vue

```diff
@@ -390,10 +390,10 @@ export default {
     ...mapState('adminAcademy', ['subjectsOfCourse', 'listCourses', 'listStudents'])
   },
   async created() {
-    await this.getCourse(this.$route.params.id);
+    let course = await this.getCourse(this.$route.params.id);
     let subjectsNoCourse = await this.getSubjectsNoCourse(this.$route.params.id);
```

Comment:
sửa cái này thành 
`let subjectsNoCourse = (await this.getSubjectsNoCourse(this.$route.params.id);).subjects`
và sau đó so sánh `subjecsNoCourse` thôi